src/main/java/core/basesyntax/Model/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.Model;
+
+import core.basesyntax.Base.Machine;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message 'Truck started its work' could be more informative. According to the checklist, it should indicate what type of Machine is working. Consider specifying the model or any other distinctive feature of the Truck to make the message clear.